test/v200/utils/common/test_utils.go

```diff
@@ -0,0 +1,283 @@
+package common
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	schema "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+	header "github.com/devfile/api/v2/pkg/devfile"
```

Comment:
pls consider not to alias it if its not ambiguous. if you really want to alias it, other files in devfile/api repo uses `workspaces` for `v1alpha2`

Reply:
I like to alias even when not ambiguos - better serviceablility. Schema fits for the tests.